test/e2e/run_volume_test.go

```diff
@@ -222,6 +222,26 @@ var _ = Describe("Podman run with volumes", func() {
 		Expect(matches[0]).To(Not(ContainSubstring("nosuid")))
 	})
 
+	// Container should start when workdir is overlayed volume
+	It("podman run with volume mounted as overlay and used as workdir", func() {
+		SkipIfRemote("Overlay volumes only work locally")
+		if os.Getenv("container") != "" {
+			Skip("Overlay mounts not supported when running in a container")
+		}
+		if rootless.IsRootless() {
+			if _, err := exec.LookPath("fuse_overlay"); err != nil {
+				Skip("Fuse-Overlayfs required for rootless overlay mount test")
```

Comment:
is this the correct name for the fuse-overlayfs binary?

I think we have the same problem in the other tests.

Reply:
@giuseppe oh i am not sure i copied it from other tests.

Reply:
@giuseppe correct name is `fuse-overlayfs` on latest f34 i think its wrong on all the existing tests but weird how tests are passing for cases where overlay driver is not present.

Reply:
yes, it is `fuse-overlayfs`

Reply:
@giuseppe It seems both are resolving correctly, i'll create a seperate PR to update the alias in entire test.

Reply:
@giuseppe updated to `fuse-overlayfs` for this test.

Reply:
thanks!